ui/js/controllers/perf/compare.js

```diff
@@ -588,6 +589,17 @@ perf.controller('CompareSubtestResultsCtrl', [
                             };
                         }), [$scope.originalResultSet, $scope.newResultSet])
                     }];
+                    cmap.links.push({
```

Comment:
We should probably only enable this for talos for now, since that's the only place this works. Could you test against the framework id and only add this if it equals 0?